test/workbox-cli/node/lib/questions/ask-root-of-web-app.js

```diff
@@ -14,8 +14,12 @@ const {errors} = require('../../../../../packages/workbox-cli/build/lib/errors')
 const MODULE_PATH = '../../../../../packages/workbox-cli/build/lib/questions/ask-root-of-web-app';
 // This is the hardcoded name of the question that's passed to inquirer.
 // It's used as the key to read the response from the answer.
-const QUESTION_NAME = 'globDirectory';
+const question_rootDirectory = 'globDirectory';
```

Comment:
Can you switch this and the other variable to `questionRootDirectory` instead of `question_rootDirectory`?

That's more in keeping with our variable naming pattern.